doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+Expected result: The field has the type `date` or `date and time`.
 
-`{"key_a" : "value_a", "key_b" : "value_b", ...}`
+Actual result: The field has the type `Any`.
 
-Actual result: The corrected output has an incorrect format (missing double quotes), as shown in the following example:
+Workaround: Manually change the type of the field to `date` or `date and time`.
 
-`{key_a : value_a, key_b : value_b, ...}`
+.When importing a data object from a Java class, a field of the Java List type is not converted to the DMN collection type [https://issues.redhat.com/browse/RHDM-1144[RHDM-1144]]
 
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type List, the resulting DMN field is not of a DMN collection type.
 
+Steps to reproduce:
 
-== Decision engine
+. Open a project in {CENTRAL}.
+. Add a Java data object.
+. Create a `List` field in this object.
+. Save and close the Java class.
+. Open a DMN file.
+. Select the *Data Types* tab.
+. Click *Import Data Object*.
+. Select the Java class that you created.
+. Click *Import*.
+. Check the type of the field into which the `List` field was converted.
 
-.The executable model does not fully parse multi-line patterns [https://issues.jboss.org/browse/RHDM-1098[RHDM-1098]]
+Expected result: The field has the type `collection`.
 
-Issue: When a pattern has multiple lines, the executable model does not fully parse its conditions. Only the first line is parsed.
+Actual result: The field has the type `Any`.
 
-NOTE: This issue is resolved with {PRODUCT} patch release 7.5.1.
+Workaround: Manually change the type of the field to `collection`.
 
-Workaround: None.
 
-endif::[]
+.When you change a decision table header in the properties panel, the change is not saved [https://issues.redhat.com/browse/RHDM-1181[RHDM-1181]]
 
-== High availability
+Issue: In the DMN designer, when editing a decision table, you can click a header to view its properties panel. However, if you change settings for the header in the properties panel, the change is not saved.
 
-//approved
-.In a high-availability authoring environment, when one user imports a project, another user is unable to see it [https://issues.jboss.org/browse/RHPAM-2470[RHPAM-2470]]
+Steps to reproduce:
 
-Issue: When multiple users connect to a high-availability {CENTRAL} and one user creates a project, another user cannot see the project.
+. Edit a decision table in an element in the DMN designer.
+. Click a header in the table.
+. Change a field, such as *Input expression*, in the properties panel on the right side of the screen.
+
+Expected result: The change is reflected in the table.
+
+Actual result: The change is not reflected in the table.
+
+Workaround: Click the header to view the pop-up window next to the header. Edit the settings in this window.
+
+.When you open a DMN diagram without layout information, nodes are all in the same position[https://issues.redhat.com/browse/RHDM-1150[RHDM-1150]]
```

Comment:
without layout information is vague

When you open a DMN diagram that does not contain layout information, ...